DetectorDescription/DDCMS/test/python/testMuonGeometry.py

```diff
@@ -19,5 +19,6 @@
 
 process.testVectors = cms.EDAnalyzer("DDTestVectors")
 process.testDump = cms.EDAnalyzer("DDTestDumpFile")
+##process.testGeoIter = cms.EDAnalyzer("DDTestDumpGeometry")
 
-process.p = cms.Path(process.test+process.testVectors+process.testDump)
+process.p = cms.Path(process.test+process.testVectors+process.testDump) ##+process.testGeoIter)
```

Comment:
@ianna out of curiosity, the new test module looks to run smoothly, I guess you comment as you use it as your optional private tests?

Reply:
@fabiocos - yes, it's just too verbose